src/Fixer/Basic/BracesFixer.php

```diff
@@ -846,6 +846,11 @@ private function ensureWhitespaceAtIndexAndIndentMultilineComment(Tokens $tokens
 
         $nextToken = $tokens[$nextTokenIndex];
         if ($nextToken->isComment()) {
+            $previousToken = $tokens[$nextTokenIndex - 1];
+            // do not indent comments starting at the beginning of line - non-indented comments should not change indentation
+            if ($previousToken->isWhitespace() && 1 === preg_match('/\R$/', $previousToken->getContent())) {
+                return;
```

Comment:
This is one of biggest fixers we have. It was introduced ages before this repo was able to properly manage dependencies between different fixers, thus to not have conflicts between them or run them in random order, it was prepared as single rule.
If you wish, go ahead! **Do** show how it shall be done and decouple it ;)